Library/Formula/vdirsyncer.rb

```diff
@@ -12,13 +12,13 @@ class Vdirsyncer < Formula
     sha256 "d0ad5bd159acdfeaf28aec0abda9f34d35750c4377624a365180c24798a8059c" => :mavericks
   end
 
-  option "without-keyring", "Build without python-keyring support"
+  option "with-remotestorage", "Build with support for remote-storage"
```

Comment:
Yeh, I think so or just remove the option (as I'm not sure I see why you'd want to disable it). Happy to either have 0.8.1 updated or wait to the next release; whichever you prefer.